compiler/erg_parser/error.rs

```diff
@@ -155,6 +155,68 @@ impl LexError {
             loc,
         ))
     }
+
+    pub fn invalid_chunk_error(errno: usize, loc: Location) -> LexError {
+        let msg = switch_lang!(
+            "japanese" => "チャンクの後に無効なチャンクが付随しています",
+            "simplified_chinese" => "",
+            "traditional_chinese" => "",
+            "english" => "Chunk is followed by invalid chunk",
```

Comment:
I wrote `chunk`, but is there a more appropriate name?

Reply:
The formal name for the smallest unit of execution, usually "each line," is chunk, which is correct. This is a loan word from Lua.
However, since it is an unfamiliar term, it seems that it would be OK to simply indicate as an invalid syntax and display a hint says that a line break or semicolon should be added.